Homebrew/Resolute Spellbook/firstdaybreak-additions.json

```diff
@@ -0,0 +1,366 @@
+        "casttime": "1 action",
+        "range": "60 feet",
+        "components": {
+            "verbal": false,
+            "somatic": false,
+            "material": ""
+        },
+        "duration": "1 round",
+        "ritual": false,
+        "description": "You psychically lash out at one creature you can see within range. The target must make an Intelligence saving throw. On a failed save, the target takes 3d6 psychic damage, and it can't take a reaction until the end of its next turn. Moreover, on its next turn, it must choose whether it gets a move, an action, or a bonus action; it gets only one of the three. On a successful save, the target takes half as much damage and suffers none of the power's other effects.\n\n**Force Potency.** When you cast this power using a force slot of 3rd level or higher, you can target one additional creature for each slot level above 2nd. The creatures must be within 30 feet of each other when you target them.",
+        "concentration": false
+    },
+    {
+        "name": "Preservation",
+        "level": 2,
+        "school": "Force",
+        "automation":[
+            {
+                "type": "text",
+                "text": "You touch a corpse or other remains. For the next 10 days, the target is protected from decay and can't become undead.\n\nThe power also effectively extends the time limit on raising the target from the dead, since days spent under the influence of this power don't count against the time limit of powers such as revitalize.\n\nAlternatively, you can touch a living creature when you cast this power. For the next hour, the target can survive within the vacuum of space. If the target breathes, it still requires oxygen to breathe. This power has no effect on dead droids or constructs."
+            }
+        ],
+        "classes": "Light",
+        "subclasses": "",
+        "casttime": "1 action",
+        "range": "Touch",
+        "components": {
+            "verbal": false,
+            "somatic": false,
+            "material": ""
+        },
+        "duration": "Special",
+        "ritual": false,
+        "description": "You touch a corpse or other remains. For the next 10 days, the target is protected from decay and can't become undead.\n\nThe power also effectively extends the time limit on raising the target from the dead, since days spent under the influence of this power don't count against the time limit of powers such as revitalize.\n\nAlternatively, you can touch a living creature when you cast this power. For the next hour, the target can survive within the vacuum of space. If the target breathes, it still requires oxygen to breathe. This power has no effect on dead droids or constructs.",
+        "concentration": false
+    },
+    {
+        "name": "Respite",
+        "level": 2,
+        "school": "Force",
+        "classes": "Light",
+        "subclasses": "",
+        "casttime": "1 bonus action",
+        "range": "60ft",

```

Comment:
No automation?

Reply:
not rn, focusing on requests first and getting the rest of the text in proper